recipe/meta.yaml

```diff
@@ -33,22 +31,22 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.31.*
+    - llvmlite 0.31.0
     - numpy
-    - tbb-devel >=2018.0.5  # [linux]
+    - tbb-devel >=2019.5  # [linux]
 
   run:
     - python
-    - {{ pin_compatible('llvmlite', max_pin='x.x') }}
-    - {{ pin_compatible('numpy') }}
+    - llvmlite >=0.31.0,<0.33
+    - {{ pin_compatible('numpy', upper_bound='1.18.0') }}
     # needed for pkg_resources
     - setuptools
 
   run_constrained:
     # If TBB is present it must be at least this version from Anaconda due to
     # build flag issues triggering UB in defaults::tbb=2018.0.4
     # See https://github.com/numba/numba/pull/3202#issuecomment-415939206 ff.
-    - tbb >=2018.0.5  # [linux]
+    - tbb >=2019.5  # [linux]
```

Comment:
This applies everywhere, Numba will `ImportError` if there's an incompatible TBB found and it's being used as the threading backend.

Reply:
Sorry I'm not totally following. What should we being doing here?

Reply:
Apologies, this wasn't very clear! I think `tbb` needs to be `>=2019.5` anywhere referenced in this recipe, e.g. https://github.com/regro-cf-autotick-bot/numba-feedstock/blob/7e2f6a21a201a06893382492831a8543c60b7c88/recipe/meta.yaml#L59 here too. That said, seems like the long running tests are being excluded in the conda forge test run so it might not matter (https://github.com/conda-forge/numba-feedstock/blob/master/recipe/run_test.sh#L39).

Reply:
Ah ok. Thanks for clarifying. I think `run_constrained` overrides that anyways (since we require a newer version). Perhaps the right thing to do is drop the version constraint there.